src/test/java/com/google/sps/SentimentServletTest.java

```diff
@@ -23,48 +28,71 @@
   @Mock
   HttpServletRequest requestMock;
 
+  @Spy
+  HttpServletResponse responseMock;
+
   /**
-   * Takes a Json String of an array and returns the elements of the array as a String, separated
-   * by periods.
-   * Ex. "["foo","bar"]" -> "foo. bar"
+   * A Json String in the form of an array should be converted to a String of period separated
+   * comments. A sentiment score should be calculated and the score should be printed in a Json
+   * object. 
+   * Ex. "foo" -> "{"commentScore":X.XX}" where X.XX is a float, for any # of decimal places. 
    */
   @Test
-  public void jsonArrayConvertsToPeriodSeparatedString() throws IOException {
+  public void jsonArrayPrintsVideoAnalysisJsonObject() throws IOException {
     String json = "[\"a\",\"b\",\"c\"]"; // "["a","b","c"]"
     when(requestMock.getReader()).thenReturn(new BufferedReader(new StringReader(json)));
     
-    String actual = sentimentServlet.requestToString(requestMock);
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter writer = new PrintWriter(stringWriter);
+    when(responseMock.getWriter()).thenReturn(writer);
```

Comment:
(1) Is this needed? Does something like `responseMock.getWriter().toString()` work?
(2) If it is needed, it seems like you need this for all your test cases so you can verify the response. You can have a method with the `@Before` attribute that automatically runs before each test. https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall